lib/instrumentation/core/http-outbound.js

```diff
@@ -196,7 +190,9 @@ function handleResponse(segment, hostname, req, res) {
   // If CAT is enabled, grab those headers!
   const agent = segment.transaction.agent
   if (agent.config.cross_application_tracer.enabled && !agent.config.distributed_tracing.enabled) {
-    pullCatHeaders(agent.config, segment, hostname, res.headers['x-newrelic-app-data'])
+    const { appData } = cat.extractCatHeaders(res.headers)
+    cat.pullCatHeaders(agent.config, segment, appData)
```

Comment:
Pull really doesn't resonate to any particular intent to me with that existing naming.

Seems like this has now been split into 'extract' which feels more appropriate and then a secondary action to grab the data from the extracted headers.

`parseCatData()` maybe?

Reply:
yea i think i'm goin to break this down even more: 
 * parseCatData
 * assignCatToSegment